app/src/main/java/com/example/dinhh/soundscape/presentation/screens/record/RecordActivity.kt

```diff
@@ -0,0 +1,123 @@
+package com.example.dinhh.soundscape.presentation.screens.record
+
+import android.Manifest
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.view.View
+import com.example.dinhh.soundscape.R
+import android.content.pm.PackageManager
+import android.media.MediaPlayer
+import android.media.MediaRecorder
+import android.os.Build
+import android.support.v4.content.ContextCompat
+import android.util.Log
+import android.widget.Toast
+import kotlinx.android.synthetic.main.activity_record.*
+import java.io.File
+import java.io.IOException
+
+class RecordActivity : AppCompatActivity() {
+
+    private val RECORD_AUDIO_REQUEST_CODE = 101
+    private var fileName: String? = null
+    private var mPlayer: MediaPlayer? = null
+    private var myAudioRecorder: MediaRecorder? = null
+
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_record)
+        btnStopRecording.visibility = View.INVISIBLE
```

Comment:
I wrote and extension function fot this. So you can use `btnStopRecording.invisible()` instead of `btnStopRecording.visibility = View.INVISIBLE`

Reply:
Fixed